libs/clients/althingi-ombudsman/src/lib/apiConfiguration.ts

```diff
@@ -0,0 +1,43 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  ConfigType,
+  IdsClientConfig,
+  XRoadConfig,
+} from '@island.is/nest/config'
+import { AlthingiOmbudsmanClientConfig } from './clients-althingi-ombudsman.config'
+import { Configuration } from '../gen/fetch/dev'
+
+export const ApiConfiguration = {
+  provide: 'AlthingiOmdudsmanClientApiConfiguration',
+  useFactory: (
+    xRoadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<typeof AlthingiOmbudsmanClientConfig>,
+    idsClientConfig: ConfigType<typeof IdsClientConfig>,
+  ) =>
+    new Configuration({
+      fetchApi: createEnhancedFetch({
+        name: 'clients-althingi-ombudsman',
+        timeout: 30000, // 30 sec timeout
```

Comment:
This timeout scares me... no user is going to wait for half a minute 🙈

Reply:
The timeout is significant due to the API entering a dormant state after inactivity. Upon reactivation, a 'wake-up' delay occurs, affecting the first user post-inactivity. We've extended the wait time instead of returning an error on application submissions, to avoid user confusion. Plus this API is only used for application submissions.